#! /bin/sh
patch -p1 -l -f $* < $0
exit $?

From 9076d3f2bcab3f8c40b2ad7806f8ae6dfa160e1c Mon Sep 17 00:00:00 2001
From: Andre Heinecke <aheinecke@gnupg.org>
Date: Mon, 19 Dec 2022 12:11:48 +0100
Subject: [PATCH] Switch to released GpgOL Version

* packages/packages.common (gpgol): Switch to released version.

--
GpgOL-2.5.6 is identical to GpgOL-2.5.5-beta5
---
 NEWS                     |  2 +-
 packages/packages.common | 13 +++----------
 2 files changed, 4 insertions(+), 11 deletions(-)

diff --git a/NEWS b/NEWS
index aed6ba40..130553db 100644
--- a/NEWS
+++ b/NEWS
@@ -148,7 +148,7 @@ Noteworthy changes in Version 4.1.0 (2022-12-20)
 ~~~~~~~~~~~~~~~
 GnuPG:          2.2.41
 Kleopatra:      3.1.26
-GpgOL:          2.5.6-beta5
+GpgOL:          2.5.6
 GpgEX:          1.0.9
 Kompendium DE:  4.0.1
 Compendium EN:  3.0.0
diff --git a/packages/packages.common b/packages/packages.common
index 5f5cfd61..e8f65667 100644
--- a/packages/packages.common
+++ b/packages/packages.common
@@ -159,11 +159,11 @@ file scute/scute-1.7.0.tar.bz2
 chk 437fe758b27c243a5ee2535c6b065ea1d09f2c9a02d83567d2f934bb6395c249
 
 # GpgOL
-# last-changed: 2022-10-14
+# last-changed: 2022-12-19
 # by: ah
 # verified: build tree
-#file gpgol/gpgol-2.5.5.tar.bz2
-#chk  d8c56b671c88e1fe9f8542bc0295b3f98202f13e81dd544dc0267c797526a75e
+file gpgol/gpgol-2.5.6.tar.bz2
+chk  f4d05b53e57fc362437236aab71c50d8e3bd441cf48986512d04dfc89faaa3c1
 
 #
 # GpgEX
@@ -406,13 +406,6 @@ server https://gnupg.org/ftp/gcrypt/snapshots
 file gpgme/gpgme-1.18.1-beta46.tar.xz
 chk 34fc73611e9081f8b2aab5431cc556bdd55493e0741426a92e276f979490f55a
 
-# gpgol
-# last changed: 2022-11-28
-# by: aheinecke
-# verified: Tarball created by aheinecke.
-file gpgol/gpgol-2.5.6-beta5.tar.xz
-chk 7ecc29787b7ea141969a6be482119051847b8aad5f492b875b874ee55f6901d3
-
 # gpgex
 # last changed: 2022-12-06
 # by: aheinecke
-- 
2.11.0

